spec/rollbar/plugins/delayed_job_spec.rb

```diff
@@ -34,6 +34,19 @@ def do_job_please!(a, b)
     end
   end
 
+  context 'with failed deserialization' do
+    let(:expected_args) do
+      [/Delayed::DeserializationError/, {:use_exception_level_filters=>true}]
+    end
+
+    it 'sends the exception' do
+      expect(Rollbar).to receive(:scope).with(kind_of(Hash)).and_call_original
+      allow_any_instance_of(Delayed::Backend::Base).to receive(:payload_object).and_raise(Delayed::DeserializationError)
```

Comment:
I guess this call to `#payload_object` is not the one we do in `JobData#handler_data` no? I say it because you are rescuing that one.